### DISCUSSION TITLE
Feature request: expose default prompts in UI

### DISCUSSION THREAD

**Discussion by willdady** [open-webui/open-webui#7024] [Ideas]
On the `Interface` tab under admin settings it's possible to override the title generation prompt but nowhere does it show what the default prompt actually is.

It would be great if these fields were pre-filled with the default prompts with the ability to edit them. There should also be a way to revert your custom change back to the default. Maybe the field is locked by default with the default prompt visible. Unlocking lets you edit it. Locking an unlocked field reverts back to the default prompt, ideally with a confirmation dialog.

**Comment by darkBuddha** [comment]
Excellent suggestion!

**Comment by Q-out** [comment]
This would be really useful right now if you wanted to run Qwen3 as the Interface Task model because you need to add a /no_think somewhere in the prompt to use it without reasoning.

**Comment by andrea-tomassi** [comment]
+1 

Exactly. or at least, expose a placeholder for the default prompt so that a user can add /nothink (or any different instruction) as a prefix/trailing part of the prompt.

**Comment by randomguyfromaplanet** [comment]
I know this is just a workaround, but you can search for GENERATION_PROMPT_TEMPLATE in the [config.py ](https://github.com/open-webui/open-webui/blob/63256136ef8322210c01c2bb322097d1ccfb8c6f/backend/open_webui/config.py) and copy and customize the prompt there.

**Reply by asahi-xd** [reply]
Thank you <3

**Comment by asahi-xd** [comment]
PLEASE do this and consider this suggestion. This will be a lifesaver to have!
